### ISSUE TITLE
Argmax giving the same result on each pixel

### ISSUE THREAD

**Issue by TimbusCalin** [bonlime/keras-deeplab-v3-plus#116]
When applying argmax on logits, the same result is for every pixel in the image.

Does anyone know how to solve this?

**Comment by TimbusCalin** [comment]
I also mention that the accuracy is very high during the training and the masks are prepared accordingly(tested with other neural network).

**Comment by graffam** [comment]
how many classes are you using, are you applying softmax or sigmoid to those logits?

**Comment by TimbusCalin** [comment]
30 classes and applying softmax not sigmoid; to the best of my knowledge sigmoid can be applied in case of overlapping classes, just like in multilabel classification.

**Comment by TimbusCalin** [comment]
Will test again with new testing configuration.
